README.md

```diff
@@ -2071,6 +2169,122 @@ Contains information about a climate control module's capabilities.
+|`heatedSteeringWheelAvailable`|Boolean|False|Availability of the control (enable/disable) of heated Steering Wheel.                True: Available, False: Not Available, Not present: Not Available.            |
+|`heatedWindshieldAvailable`|Boolean|False|Availability of the control (enable/disable) of heated Windshield.                True: Available, False: Not Available, Not present: Not Available.            |
+|`heatedRearWindowAvailable`|Boolean|False|Availability of the control (enable/disable) of heated Rear Window.                True: Available, False: Not Available, Not present: Not Available.            |
+|`heatedMirrorsAvailable`|Boolean|False|Availability of the control (enable/disable) of heated Mirrors.                True: Available, False: Not Available, Not present: Not Available.            |
+
+
+### EqualizerSettings
+Defines the each Equalizer channel settings.
+
+##### Parameters
+
+| Value |  Type | Mandatory | Description | 
+| ---------- | ---------- |:-----------: |:-----------:|
+|`channelId`|Integer|True||
+|`channelName`|String|False|read-only channel / frequency name (e.i. "Treble, Midrange, Bass" or "125 Hz")|
+|`channelSetting`|Integer|True||
+
+
+### AudioControlData
+##### Parameters
+
+| Value |  Type | Mandatory | Description | 
+| ---------- | ---------- |:-----------: |:-----------:|
+|`source`|PrimaryAudioSource|False|In a getter response or a notification, it is the current primary audio source of the system.                In a setter request, it is the target audio source that the system shall switch to.                If the value is MOBILE_APP, the system shall switch to the mobile media app that issues the setter RPC.            |
+|`keepContext`|Boolean|False|This parameter shall not be present in any getter responses or notifications.                This parameter is optional in a setter request. The default value is false.                If it is false, the system not only changes the audio source but also brings the default application                or system UI associated with the audio source to foreground.                If it is true, the system changes the audio source, but still keeps the current application in foreground.            |
+|`volume`|Integer|False|Reflects the volume of audio, from 0%-100%.|
+|`equalizerSettings`|EqualizerSettings[]|False|Defines the list of supported channels (band) and their current/desired settings on HMI|
+
+
+### AudioControlCapabilities
+##### Parameters
+
+| Value |  Type | Mandatory | Description | 
+| ---------- | ---------- |:-----------: |:-----------:|
+|`moduleName`|String|True|The short friendly name of the light control module.                It should not be used to identify a module by mobile application.            |
+|`sourceAvailable`|Boolean|False|Availability of the control of audio source. |
+|`keepContextAvailable`|Boolean|False|Availability of the keepContext paramter. |
```

Comment:
This is not included in [SDL 0099](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0099-new-remote-control-modules-and-parameters.md) or [SDL 0182](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0182-audio-source-am-fm-xm.md).  Please advise if this is part of a different proposal, or if it should be removed.